Formula/mdless.rb

```diff
@@ -0,0 +1,33 @@
+class Mdless < Formula
+  desc "Provides a formatted and highlighted view of Markdown files in Terminal"
+  homepage "https://github.com/ttscoff/mdless"
+  url "https://github.com/ttscoff/mdless/archive/refs/tags/1.0.32.tar.gz"
+  sha256 "ce67a184aaeba66955c96f9f2d3353040b359c166329a149c49b470bf8edeb39"
+  license "MIT"
+
+  uses_from_macos "ruby"
+
+  def install
+    ENV["GEM_HOME"] = libexec
+    system "gem", "build", "#{name}.gemspec"
+    system "gem", "install", "#{name}-#{version}.gem"
+    bin.install libexec/"bin/#{name}"
+    bin.env_script_all_files(libexec/"bin", GEM_HOME: ENV["GEM_HOME"])
+  end
+
+  test do
+    assert_match "mdless #{version}", shell_output("#{bin}/mdless --version")
+    (testpath/"test.md").write <<~EOS
+      # title first level
+      ## title second level
+    EOS
+    shell_output("#{bin}/mdless --no-color -P test.md").chomp.split("\n").each do |line|
+      if line.start_with?("title first level")
+        assert line.end_with?("==="), "first level title expected to use === delimiter"
+      end
+      if line.start_with?("title second level")
+        assert line.end_with?("---"), "second level title expected to use --- delimiter"
+      end
+    end
```

Comment:
```suggestion
    out =  shell_output("#{bin}/mdless --no-color -P test.md")
    assert_match /^title first level .* ===$/, out
    assert_match /^title second level .* ---$/, out
```

Reply:
Ah! I wasn't aware `assert_match` supported multi-line matching. Thanks for the suggestion @SMillerDev. Gonna adapt it a bit and update the block

Reply:
Done! ([d93699d](https://github.com/Homebrew/homebrew-core/pull/121410/commits/d93699d54b8d3001b10155dcf913d6a6833b065c))

Reply:
Completed by [2540f94](https://github.com/Homebrew/homebrew-core/pull/121410/commits/2540f94d8757d3e750fff9942d09cdb3bd3ae6d0)